Makefile

```diff
@@ -27,3 +36,25 @@ install:
 
 test:
 	panther_analysis_tool test
+
+managed-schemas:
```

Comment:
@alxarch I'd prefer if we used the `panther_analysis_tool` for this functionality

Reply:
Not really possible right now. We rely on this on `panther` repo as well. I would be quite complex to coordinate. We an revisit this when we take a closer look on the automations. AFAIK pat is Python so it will be tricky to do the automations there since tests and validation all rely on Go code. Bash glues everything nicely.

Reply:
I'm not following - I'm saying use Python for zipping and hashing instead of bash.

Reply:
I am already stretched thin having to coordinate between 2 repositories for an under-development feature with many moving parts. We can explore moving this to the pat repository once we have an MVP working.